### PR TITLE
Evan/display my teams cards

### PR DESCRIPTION
**Pull request by evazhog**
Solves #13 and #23 

When a user views the teams screen, our app will pull all teams that user is in and display them in pills:
image

When a user clicks on a pill for a specific team, they will see the "SpecificTeamScreen"
image

Still WIP:
-Getting the bunny / animal icon to be positioned correctly. (problem has to do with the automatic 16px padding from the children wrapper in `KitBackgroundScreen`)
-Getting the "past challenges" pills to be interactive, awaiting some work on challenges / challenge object structure

### CODE REVIEW

**Comment by biancavillamor** [comment]
hi so i made a change on my branch that fixed the weird padding on the header and child (i made a prop that could change the padding, which i changed to 1, and then made paddingTop: 16 for the header so that it would still be positioned correctly) and hopefully when i push then merge than change u could use that to reposition the bunny??

File: components/team-codes/MyTeams.js [hunk 1]
```diff
@@ -110,3 +125,7 @@ const styles = StyleSheet.create({
     justifyContent: "center"
   }
 });
+
+//Calling withNavigation() because props.navigation is only accessible for Screens
+export default withNavigation(MyTeams);
```
Comment (kristielim) [inline components/team-codes/MyTeams.js]: You could also just move all of MyTeams to TeamsScreen. Up to you though

File: components/team-codes/MyTeams.js [hunk 2]
```diff
@@ -95,7 +100,17 @@ const styles = StyleSheet.create({
...
+  },
+  teamBarFill: {
+    color: "#F6F6F6", //EVAN TODO: ask design if they're really set on this color or if we can use other constants
+    backgroundColor: Colors.KIT_ORANGE, 
+  },
+  teamBarNoFill: {
+    borderWidth: 1,
+    borderStyle: "solid",
+    borderColor: "#D65044", //EVAN TODO: ask design if they're really set on this color or if we can use other constants
+    color: "#3A3A3A", //EVAN TODO: ask design if they're really set on this color or if we can use other constants
```
[the remark this answers is not in the document]
Comment (kristielim) [reply components/team-codes/MyTeams.js]: But also I say you can also make the executive decision just to use the most similar color

[the remark this answers is not in the document]
Comment (evazhog) [reply components/team-codes/MyTeams.js]: I'm going to merge it as it with these hardcoded colors, b/c I know you've made some updates to `Color.js` and don't want to use an out of date color.

I'll sort out these hardcoded instances on the next iteration of this, resolving other WIP tasks.

**Pull request merged**